src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java

```diff
@@ -103,6 +108,22 @@ public void setInstitute(String institute) {
         this.institute = institute;
     }
 
+    public ReadNotifications getReadNotifications() {
+        return readNotifications;
+    }
+
+    public void setReadNotifications(ReadNotifications readNotifications) {
+        this.readNotifications = readNotifications.getDeepCopy();
+    }
+
+    public ReadNotifications getReadNotificationsCopy() {
+        return readNotifications.getDeepCopy();
+    }
+
+    public String getReadNotificationsAsText() {
+        return readNotifications.getJsonString();
+    }
+
```

Comment:
In the method of `getInvalidityInfo()` below (line135-150), do we need to add a check for `readNotifications`?

Reply:
You're right, it would be good to add validity checks for this, I'll work on it!

Reply:
Upon closer inspection, I think we should not add validity checks here as the updating of the `readNotifications` attribute will be handled by the system. Returning error messages here may cause confusion to the user.
If need be, we can come back to this after adding the API/frontend to make the error checks consistent